segway/gmtk/params.py

```diff
@@ -282,8 +385,7 @@ class MC:
     def __init__(self, component_type: str):
         """
         Initialize a single MC object.
-        :param component_type: str: type of MC, such as
-            COMPONENT_TYPE_DIAG_GAUSSIAN
+        :param component_type: str: type of MC
```

Comment:
add examples

Reply:
Added `"COMPONENT_TYPE_DIAG_GAUSSIAN"` and `"COMPONENT_TYPE_MISSING_FEATURE_SCALED_DIAG_GAUSSIAN"` as examples.